assets/src/components/SurveyModal.js

```diff
@@ -50,8 +51,25 @@ const useStyles = makeStyles((theme) => ({
 
 export default function SurveyModal (props) {
   const classes = useStyles()
+  const location = useLocation()
   const [open, setOpen] = useState(false)
 
+  const lastTwoSteps = location.pathname.split('/').slice(-2)
+  const viewName = lastTwoSteps.length < 2
+    ? ''
```

Comment:
This was just a fallback (maybe it should be `unknown`?). I don't love this approach since it could easily break if you changed the application structure, but that seems unlikely to happen for now.

I guess the alternative would be passing hardcoded values as props wherever `DashboardAppBar` is used. That could be more flexible, but would require updating a handful more files.

Reply:
Alternatively, we could just pass the whole `pathname`.

Reply:
yeah, something like `location.pathname || 'unknown'`.

Reply:
That's not important, though.  It'd be nice to have, but it can be changed in a future version if it's decided that's a better approach.